src/notation/internal/notationparts.cpp

```diff
@@ -37,15 +37,39 @@ using namespace mu::engraving;
 
 static const mu::engraving::Fraction DEFAULT_TICK = mu::engraving::Fraction(0, 1);
 
+static QString formatInstrumentTitleOnScore(const QString& instrumentName, const Trait& trait)
+{
+    // Comments for translators start with //:
+
+    if (trait.type == TraitType::Transposition && !trait.isHiddenOnScore) {
+        //: %1=name ("Horn"), %2=transposition ("C alto"). Example: "Horn in C alto"
+        return mu::qtrc("notation", "%1 in %2", "Transposing instrument displayed in the score")
```

Comment:
Keep in mind that the "disambiguation" needs to be repeated exactly in all places where we want to use the same translatable string. Therefore, in general, I don't consider it a good place to add long comments, rather for a few simple words that are not prone to typos. 
However, since it is very unlikely that we will ever use these specific strings in other places, I'd say this is okay for now.

Reply:
I think a shorter disambiguation string would require a longer translation comment to explain what it means. I'll experiment with `lupdate` to see whether/how it handles multi-line translation comments.

Reply:
My experience is that `lupdate` strips newlines from `//:` comments. 

Anyway, since each of these strings is used in only one place for now, I'm happy with it as it is, but it's just something to keep in mind for other cases.